src/Microsoft.Health.SqlServer/Configs/SqlServerDataStoreConfiguration.cs

```diff
@@ -51,5 +51,10 @@ public class SqlServerDataStoreConfiguration
         /// If set, the client id of the managed identity to use when connecting to SQL, if AuthenticationType == ManagedIdentity.
         /// </summary>
         public string ManagedIdentityClientId { get; set; }
+
+        /// <summary>
+        /// Specifies the statement timeout to set on ServerConnection.
+        /// </summary>
+        public int StatementTimeoutInSecs { get; set; } = 14400;
```

Comment:
I think a TimeSpan would be better, but check if that works with config as expected

Reply:
Thanks, updated!